static/app/views/stories/storySourceLinks.tsx

```diff
@@ -55,3 +56,8 @@ export function StorySourceLinks(props: {story: StoryDescriptor}) {
     </Fragment>
   );
 }
+
+const LastEdited = styled('span')`
+  font-size: ${p => p.theme.fontSizeSmall};
+  color: ${p => p.theme.subText};
```

Comment:
we should dogfood our own tokens I guess 😂 

```suggestion
  color: ${p => p.theme.tokens.content.muted};
```